docs/csharp/programming-guide/unsafe-code-pointers/pointer-types.md

```diff
@@ -109,7 +109,7 @@ Console.ReadLine();
   
  A pointer can be `null`. Applying the indirection operator to a null pointer causes an implementation-defined behavior.  
   
- Be aware that passing pointers between methods can cause undefined behavior. Examples are returning a pointer to a local variable through an Out or Ref parameter or as the function result. If the pointer was set in a fixed block, the variable to which it points may no longer be fixed.  
+ Be aware that passing pointers between methods can cause undefined behavior. Examples are returning a pointer to a local variable through an In, Out or Ref parameter or as the function result. If the pointer was set in a fixed block, the variable to which it points may no longer be fixed.  
```

Comment:
unrelated to your PR: are returning --> return
it would be best to lowercase in, out, and ref, and possibly fence them.